New post: IRaaS: variants and scope (#22)
mmo/sonar.ch: content/post/iraas_service.md

---
title: "Institutional Repository as a Service, by RERO: variants and scope"
date: 2020-06-04T08:00:00+02:00
draft: false
tags: ["iraas", "repository", "shared portal", "dedicated portal"]
---

The project SONAR has been raising a growing interest within the Swiss scientific community. Especially small and medium sized Higher Education Institutions (HEI) showed their interest for the Institutional Repository as a Service (short IRaaS).

> The service is not yet available, but the RERO central and its project partners are working hard to implement it. However, we could take some time to define the scope of IRaaS, and its two possible variants: the shared portal and the dedicated portal. You find all the details on the [IR hosting](/iraas) page.

Reviewer: ```suggestion
The service is not yet available, but the RERO central and its project partners are working hard to implement it. However, we can take some time to define the scope of IRaaS, and its two possible variants: the shared portal and the dedicated portal. You'll find all the details on the [IR hosting](/iraas) page.
```
Maybe I do not understand what you mean, it's just the way I would have put it.

Author: I finally put "we took some time...". I wanted to say "Nous avons pu prendre le temps..."
